Userland/Demos/Fire/Fire.cpp

```diff
@@ -134,7 +134,7 @@ void Fire::timer_event(Core::TimerEvent&)
     if (phase > 1)
         phase = 0;
 
-    /* Paint our palettized buffer to screen */
+    /* Paint our palletized buffer to screen */
```

Comment:
That'sa mis-correction? <https://en.wiktionary.org/wiki/palettize>

Reply:
Funnily enough, this was already merged. So this PR should probably revert that change.